### PR TITLE
The project itself

### PR DESCRIPTION
**Pull request by Yarin18**

### CODE REVIEW

**Comment by DanneD-Wisemen** [comment]
Don't forget to add @Jesse-Appwise as a reviewer

File: composeApp/src/androidMain/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@android:style/Theme.Material.Light.NoActionBar">
+        <activity
+            android:exported="true"
+            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden|mnc|colorMode|density|fontScale|fontWeightAdjustment|keyboard|layoutDirection|locale|mcc|navigation|smallestScreenSize|touchscreen|uiMode"
```
Comment (DanneD-Wisemen) [inline composeApp/src/androidMain/AndroidManifest.xml:13]: Are all these configChanges necessary for kmp? Just a question 😅

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```
Comment (DanneD-Wisemen) [inline gradle/wrapper/gradle-wrapper.properties:3]: Just a question: here you use the 8.4 gradle version and in the version catalog 8.2.0 is used. Does this conflict and was this a choice?

File: shared/src/androidMain/kotlin/Platform.android.kt
Comment (DanneD-Wisemen) [inline shared/src/androidMain/kotlin/Platform.android.kt:1]: I think this file could be removed

File: shared/src/commonMain/kotlin/Greeting.kt
Comment (DanneD-Wisemen) [inline shared/src/commonMain/kotlin/Greeting.kt:1]: I think this file could be removed

File: shared/src/commonMain/kotlin/Platform.kt
Comment (DanneD-Wisemen) [inline shared/src/commonMain/kotlin/Platform.kt:1]: I think this file could be removed

File: shared/src/iosMain/kotlin/Platform.ios.kt
Comment (DanneD-Wisemen) [inline shared/src/iosMain/kotlin/Platform.ios.kt:1]: I think this file could be removed

**Review by DanneD-Wisemen — CHANGES_REQUESTED** [review]

File: composeApp/src/androidMain/AndroidManifest.xml (hunk 1 above)
Comment (Yarin18) [reply composeApp/src/androidMain/AndroidManifest.xml:13]: I did not touch this file, this was automatic as far as I am aware

Comment (Yarin18) [reply composeApp/src/androidMain/AndroidManifest.xml:13]: I did not touch this file, this is automatic as far as Im aware

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (Yarin18) [reply gradle/wrapper/gradle-wrapper.properties:3]: This does not seem to conflict, also not a choice, it was autogenerated like this. The highest I could bump it to in the version catalog is 8.2.2 which I just did, not sure why 8.4 can't be found

File: shared/src/commonMain/kotlin/Greeting.kt
Comment (Yarin18) [reply shared/src/commonMain/kotlin/Greeting.kt:1]: I kept this in to make a small example for Koin injection in swift, (PR 2), would it be better to keep this in, or do you want me to remove it?

Comment (DanneD-Wisemen) [reply shared/src/commonMain/kotlin/Greeting.kt:1]: If you think you need the example for now, you can keep it

**Review by DanneD-Wisemen — APPROVED** [review]

**Pull request merged**